docs/_chapters/125-tour-features.md

```diff
@@ -66,7 +66,7 @@ Since bnd files are property files, you cannot repeat a property. Later properti
 
 We can now wrap the source JAR with the following command:
 
-	$ bnd javax.activation.bnd
+	$ bnd javax.activation.jar
 
```

Comment:
This change is incorrect. The line was correct before your change. The bnd file was created at the end of `Simplistic` above.